src/InfluxDB2/Client.php

```diff
@@ -118,18 +134,21 @@ public function getConfiguration()
      */
     public function createService($serviceClass)
     {
+        if (!($this->getApi() instanceof GuzzleApi)) {
+            throw new RuntimeException('Only Guzzle API is supported!');
+        }
+
         try {
             $class = new ReflectionClass($serviceClass);
             $args = array($this->getGuzzleClient(), $this->getConfiguration());
```

Comment:
We should use existing instance of `GuzzleApi`:

```suggestion
            $args = array($this->getApi(), $this->getConfiguration());
```